lib/pbench/server/database/models/api_keys.py

```diff
@@ -47,8 +47,10 @@ class APIKey(Database.Base):
     """Model for storing the API key associated with a user."""
 
     __tablename__ = "api_keys"
-    api_key = Column(String(500), primary_key=True)
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    api_key = Column(String(500), unique=True, nullable=False)
     created = Column(TZDateTime, nullable=False, default=TZDateTime.current_time)
+    name = Column(String(128), unique=False, nullable=False)
```

Comment:
`unique=False, nullable=False` is the default, so these could be omitted... although it doesn't wrap the line and I'm not opposed to the extra documentation. 😁

Reply:
Or, if we go with Webb's opinion, `nullable=True`. I guess we need to decide on this. Especially if we're not going *display* the `id` column in the UI, I like the idea of something which, by default, suggests the sequence; as in, without names, "the first key I created" vs "the second key I created", so something like `API_KEY_1` might work. (But do we use `id` for the numeral, which is unique but, for an individual user, non-sequential, or do we invent something else? And even think of that makes me think maybe just leave it blank and nullable and if they don't supply a name that's on them...)

Reply:
> if we're not going display the id column in the UI, I like the idea of something which, by default, suggests the sequence

Just because we don't display the ID doesn't mean that the UI cannot use it to order the entries in the table.  We're expecting the ID to be the table entry key, perhaps it will order the entries by default?

Also, I think we should have the creation date displayed in the table (e.g., this enables user policies about recycling keys based on age), which will yield the creation sequence for a user.

> do we use id for the numeral, which is unique but, for an individual user, non-sequential, or do we invent something else?

I'm not excited about inventing things which bring so little bang for our buck.  (I don't think non-sequential numbers in default description text is likely to be a problem...unlike non-sequential numbers in table indices.)  Leaving it blank (until we have a requirement to do something different) seems like a good approach.